NuGet.config

```diff
@@ -3,5 +3,9 @@
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
Yes, there is: https://github.com/aspnet/Universe/issues/362

Feel free to add any extra packages to the list